scripts/linters/pylint_extensions.py

```diff
@@ -488,6 +560,44 @@ class DocstringParameterChecker(checkers.BaseChecker):
     not_needed_param_in_docstring = {'self', 'cls'}
     docstring_sections = {'Raises:', 'Returns:', 'Yields:'}
 
+    # Docstring section headers split up into arguments, returns, yields and
+    # and raises sections signifying that we are currently parsing the
```

Comment:
Duplicate "and". OK to fix in next PR, but please keep track. /cc @Hudda @bansalnitish

Reply:
Okay, I will fix this in my next PR.

Reply:
Please file an issue for this and the next comment, and assign it to yourself (standard procedure for any deferred changes). Once you post the issue link here, I will merge this PR.

Reply:
I've created the issue https://github.com/oppia/oppia/issues/10102